tests/mints9/purepythonanalog.in

```diff
@@ -13,15 +13,15 @@ H_l -0.5  0.7 0.0
 """)
 
 print('[1]    <<<  uniform cc-pVDZ  >>>')
-wert = qcdb.BasisSet.pyconstruct(mymol, 'BASIS', 'cc-pvdz')
+wert = qcdb.BasisSet.pyconstruct(mymol, 'BASIS', 'cc-pvdz')[0]
 compare_integers(38, wert.nbf(), 'nbf()')
 compare_integers(40, wert.nao(), 'nao()')
 compare_strings('c2v', mymol.schoenflies_symbol(), 'symm')
 mymol.print_out()
 
 
 print('[2]        <<<  RIFIT (default)  >>>')
-wert = qcdb.BasisSet.pyconstruct(mymol, 'DF_BASIS_MP2', '', 'RIFIT', 'cc-pvdz')
+wert = qcdb.BasisSet.pyconstruct(mymol, 'DF_BASIS_MP2', '', 'RIFIT', 'cc-pvdz')[0]
 qcdb.compare_integers(140, wert.nbf(), 'nbf()')
```

Comment:
All these '[0]'s are an unintended consequence of my hacking on the BasisSet constructor py-side.  I thought the design was harmless at the time, so sorry for the extra hassle.  It'll all be cleaned up next week, hopefully.